### PR TITLE
UI Plugin: Move and rename some the MC create workflow components

### PR DESCRIPTION
**Pull request by miclettej**
## What this PR does / why we need it
Moves the AWS and Docker MC create workflows into the views folder

Renamed some of these files to be more representative of their purpose, but open to alternative suggestions

## Which issue(s) this PR fixes

Fixes: n/a

## Describe testing done for PR
confirmed UI still runs as expected, no new linting/test errors introduced

### CODE REVIEW

File: cli/cmd/plugin/ui/web/tanzu-ui/src/App.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { Routes, Route } from 'react-router-dom';
 import { APP_ENV_CHANGE, AppActionNames } from './state-management/actions/App.actions';
 import { NavRoutes } from './shared/constants/NavRoutes.constants';
 import { Store } from './state-management/stores/Store';
-import AwsManagementClusterSimple from './views/management-cluster/AwsManagementClusterSimple';
```
Comment (swalner-vmware) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/App.tsx:9]: I thought we were using `Simple` here to distinguish from our anticipated `Advanced` flow in the future. (Maybe `Basic` is better?)

File: cli/cmd/plugin/ui/web/tanzu-ui/src/App.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ import UnmanagedClusterLanding from './views/unmanaged-cluster/UnmanagedClusterL
 import VSphere from './components/VSphere';
 import Welcome from './views/welcome/Welcome';
 import WorkloadClusterWorkflow from './views/workload-cluster/WorkloadClusterWorkflow';
-import DockerManagementClusterSimple from './views/management-cluster/DockerManagementClusterSimple';
```
Comment (swalner-vmware) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/App.tsx:19]: Likewise `Simple` vs `Advanced` and perhaps `Basic` instead?

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/wizard-simple/AwsManagementClusterSimple.tsx [hunk 3]
```diff
@@ -2,14 +2,14 @@
 import React, { useContext } from 'react';
 
 // App imports
-import { AwsStore } from '../../state-management/stores/Store.aws';
-import { TAB_NAMES } from '../../shared/constants/NavRoutes.constants';
-import Wizard from '../../shared/components/wizard/Wizard';
-import ManagementClusterSettings from '../../shared/components/management-cluster-settings-step/ManagementClusterSettings';
+import { AwsStore } from '../../../../state-management/stores/Store.aws';
+import { TAB_NAMES } from '../../../../shared/constants/NavRoutes.constants';
```
Comment (swalner-vmware) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/wizard-simple/AwsManagementClusterSimple.tsx:6]: I'm not sure it really makes sense to include all the workflow tabs names in a single TAB_NAMES array. Feels like each workflow should define their own

**Review by swalner-vmware — APPROVED** [review]

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/wizard-simple/AwsManagementClusterSimple.tsx (hunk 3 above)
Comment (miclettej) [reply cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/wizard-simple/AwsManagementClusterSimple.tsx:6]: +1, will update this

File: cli/cmd/plugin/ui/web/tanzu-ui/src/App.tsx (hunk 2 above)
Comment (miclettej) [reply cli/cmd/plugin/ui/web/tanzu-ui/src/App.tsx:19]: Agreed, basic sounds better

**Pull request merged**